iron-autogrow-textarea.html

```diff
@@ -167,6 +168,15 @@
       },
 
       /**
+       * The value for this input, same as `bindValue`
+       */
+      value: {
+        notify: true,
+        type: String,
+        computed: '_computeValue(bindValue)'
```

Comment:
Perhaps although that would entail a breaking API change. Might be better to keep backwards compatible.